examples/example-imgui.hpp

```diff
@@ -47,10 +48,28 @@ class hdr_slider : public slider {
         ImGui::SetNextWindowPos(_position);
         //concate the name given with seq_id in order to make a unique name (uniqeness is needed for Begin())
         std::string name_id = std::string(_name) + std::to_string(_seq_id);
+        ImGuiStyle& style = ImGui::GetStyle();
+        ImVec4* colors = style.Colors;
```

Comment:
Do we set the same colors to all examples?
Maybe we can add a helper function for it?

Reply:
added the helper function ImGuiSetStyleColors in example.hpp